core/parameters/res/runtime_configs/87.yaml

```diff
@@ -2,8 +2,11 @@
 # State Witness
 
 max_transaction_size: {old: 4_194_304, new: 1_572_864}
+max_receipt_size: {old: 999_999_999_999_999, new: 4_194_304}
```

Comment:
It might be possible to lower the receipt limit further, but it would break some receipts, there are some 3.9MB ones. It would require some more thinking about, for now 4MiB seems like a safe and good enough choice, we can lower it in a follow-up PR.
https://near.zulipchat.com/#narrow/stream/295306-contract-runtime/topic/How.20large.20can.20internal.20receipts.20get.3F/near/443309601